### DISCUSSION TITLE
Add list of figures/tables to a word document

### DISCUSSION THREAD

**Discussion by alchalu** [quarto-dev/quarto-cli#1564] [Q&A]
Hello, 

is there any possibility to add a list of figures or a list of tables to a word document created with Quarto?

**Comment by rgaiacs** [comment]
From the documentation, list of figures or a list of tables are only available to PDF.

**Comment by cscheid** [comment]
As @rgaiacs mentioned, we don't currently have this functionality, but it's a very good idea. No timeline for it yet, but I created #1600 to track.

**Reply by larnsce** [reply]
Thank you for adding it, @cscheid. When teaching Quarto (mostly in academia), it's the most common question, but for HTML and not necessarily DOCX. It's a requirement for anyone submitting a research report or thesis and the capability would increase the potential for Quarto to be used more commonly for such reports.

**Reply by cscheid** [reply]
@larnsce I've been a professor for 8 years of my life. We're aware of that! There's just a lot else on our plates. We appreciate the patience!

**Reply by larnsce** [reply]
@cscheid That must have come across in a way I didn't intend. I apologise for that. I didn't mean to add any pressure as I am aware you have a lot going on and appreciate the contributions you make with your open-source software development.

**Reply by cscheid** [reply]
No worries! I just wanted to ensure that "yes, I really do wish we had it, and I _definitely_ want to work on it. But the todo lists are big right now :)"

**Comment by mcanouil** [comment]
To note, "Table of Figures" (it's even in the screenshot of <https://quarto.org/docs/output-formats/ms-word-templates.html>) is a standard/default theme component in Word, thus one can define its own template to have it by default, at least in theory (I did not try it yet).

From the reply[^1] I got on Twitter when I suggested the template approach, it seems to work.


[^1]: <https://twitter.com/Brier_Gallihugh/status/1582106787548868608>
